app/javascript/components/track/approaches-elements/community-videos/CommunityVideo.tsx

```diff
@@ -24,7 +24,7 @@ export function CommunityVideo({
             alt="thumbnail"
           />
           <div className="flex flex-col">
-            <h5 className="text-h5 mb-8">{video.title}</h5>
+            <h5 className="text-h5 mb-8 text-left">{video.title}</h5>
```

Comment:
@dem4ron Here's a loom on this bit of code in general: https://www.loom.com/share/0901d0e13a4242cca67ccd15a17ced93?from_recorder=1&focus_title=1

Reply:
Yeah, I agree, this was a mess.. I started using a lot more grid for this reason, but that can become overcomplicated in Tailwind without template-areas support..